exporters/stdout/stdouttrace/example_test.go

```diff
@@ -77,11 +78,11 @@ func InstallExportPipeline(ctx context.Context) (func(context.Context) error, er
 	return tracerProvider.Shutdown, nil
 }
 
-func Example() {
+func TestExample(t *testing.T) {
```

Comment:
This transforms the example test into a test. The name should not be modified.

https://go.dev/blog/examples